pytest/tests/mocknet/mirror.py

```diff
@@ -393,6 +398,9 @@ def start_nodes_cmd(args, traffic_generator, nodes):
 
 
 def start_traffic_cmd(args, traffic_generator, nodes):
+    if traffic_generator is None:
+        logger.warning('No traffic node selected. Change filters.')
```

Comment:
if only traffic node is selected, could it happen that the nodes list is empty in the rest of this function? (would it block the wait operation below)?

Reply:
yes, but then they can call `start-nodes` if that was the user's intention

Reply:
I misread your question. If the node list is empty, the traffic node will start and it will not wait for the other nodes as none were provided.